src/engine/sparqlExpressions/SparqlExpressionValueGetters.cpp

```diff
@@ -37,6 +38,14 @@ bool EffectiveBooleanValueGetter::operator()(ValueId id,
       return id.getInt() != 0;
     case Datatype::Undefined:
       return false;
+    case Datatype::WordVocabIndex: {
+      auto index = id.getVocabIndex();
```

Comment:
That is really unfortunaate. The `textVocab` should store `WordVocabIndex` and not `VocabIndex`. I'll have a look at how big of a change this would be. It technically works, but it is really unfortunate.